tests/core/test_full_node_rpc.py

```diff
@@ -195,6 +195,10 @@ def stop_node_cb():
             assert len(await client.get_coin_records_by_puzzle_hash(ph, True, 0, blocks[-1].height + 1)) == 2
             assert len(await client.get_coin_records_by_puzzle_hash(ph, True, 0, 1)) == 0
 
+            block_spends = await client.get_block_spends((await full_node_api_1.get_all_full_blocks())[-1].header_hash)
+
+            assert block_spends == spend_bundle.coin_spends
```

Comment:
it's not obvious that `blocks[-1]` is a very interesting block to test. I would have expected this test for *every* block